apis/r/configure

```diff
@@ -36,6 +36,13 @@ else
         rm -rf build
         cd ..
         echo "** static library made"
+
+        ## handle different distro library install paths
+        if test -d src/lib64; then
+            mv src/lib64 src/lib
+        elif test -d src/lib/x86_64-linux-gnu; then
+            mv src/libx86_64-linux-gnu src/lib
+        fi
```

Comment:
Yes likely the best options is to pass `-DCMAKE_INSTALL_LIBDIR=lib` to the `cmake` command as documented [here](https://cmake.org/cmake/help/v3.15/module/GNUInstallDirs.html). I encountered issues when trying this (i.e. it still kept intalling into `lib64` when I tried it on a RHEL9 machine), but this could be debugged in a future attempt.